api/identifiers/views.py

```diff
@@ -14,12 +14,14 @@
 from api.nodes.permissions import (
     IsPublic,
     ExcludeWithdrawals,
+    AdminOrPublic,
+    EditIfPublic,
 )
 
 from osf.models import Node, Registration, Preprint, Identifier
 
 
-class IdentifierList(JSONAPIBaseView, generics.ListAPIView, ListFilterMixin):
+class IdentifierList(JSONAPIBaseView, generics.ListCreateAPIView, ListFilterMixin):
     """List of identifiers for a specified node. *Read-only*.
```

Comment:
Good point @aaxelb, I'm deleting it because this is just a base view; the appropriate dev docs are already linked on the respective NodeIdentifiersList and RegistrationIdentifiersList views.

I've written docs for creating DOI's - will link them here when I get a PR up.

Reply:
Additionally, I added the IDENTIFIERS_WRITE scope with my latest commit.